src/test/scala/org/constellation/cluster/E2ETest.scala

```diff
@@ -46,6 +47,15 @@ class E2ETest extends AsyncFlatSpecLike with Matchers with BeforeAndAfterAll wit
 
     TestNode(randomizePorts = randomizePorts, portOffset = portOffset, seedHosts = seedHosts)
   }
+  val updatePasswordReq = UpdatePassword("updatedPassword")
+
+  def updatePasswords(apiClients: Seq[APIClient]): Seq[APIClient] = {
+    apiClients.foreach { client =>
+      client.postSync("password/update", updatePasswordReq)
```

Comment:
you should verify that the call worked. Check the status code or assert .isSuccess on the returned response object.

Reply:
👍